scripts/integration-tests/test-curiosity.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eu
+
+if [[ -z "${CURIOSITY_ENDPOINT:-}" ]]; then
+    scriptdir=$(realpath -s "$0" | xargs dirname)
+    echo "Missing curiosity endpoint."
+    echo "Did you read $scriptdir/readme.md ?"
+    exit 1
+fi
+
+echo "[+] Querying curiosity root endpoint"
+curl -f -v "$CURIOSITY_ENDPOINT" > /dev/null
```

Comment:
From a quick look at the output of `nix-build -A run-vm-test`, I think you could add `--silent` to curl options.

Reply:
I find it convenient to see the detailed failure output when the test fails on CI (hence the `-v` flag). It saves you a re-run just to get the logs.

Of course, when it succeeds, the output is not that interesting :)

Reply:
I think `-s` and `-v` can be combined. I think the `-s` will get rid of the progression bar.

Reply:
Are you sure? From the manual:

```
       -s, --silent
              Silent  or  quiet mode. Do not show progress meter or error messages. Makes Curl mute. It will
              still output the data you ask for, potentially even to the terminal/stdout unless you redirect
              it.
```

Reply:
Oh yeah, you were talking about errors. I was after this:

```
[thu@tank:~/projects/tvl-depot]$ curl -s -v 127.0.0.1:9000/nix-cache-info > /dev/null
*   Trying 127.0.0.1:9000...
* Connected to 127.0.0.1 (127.0.0.1) port 9000 (#0)
> GET /nix-cache-info HTTP/1.1
> Host: 127.0.0.1:9000
> User-Agent: curl/7.83.1
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Date: Mon, 21 Nov 2022 14:37:28 GMT
< Content-Length: 51
< Content-Type: text/plain; charset=utf-8
< 
{ [51 bytes data]
* Connection #0 to host 127.0.0.1 left intact

[thu@tank:~/projects/tvl-depot]$ curl -v 127.0.0.1:9000/nix-cache-info > /dev/null
*   Trying 127.0.0.1:9000...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0* Connected to 127.0.0.1 (127.0.0.1) port 9000 (#0)
> GET /nix-cache-info HTTP/1.1
> Host: 127.0.0.1:9000
> User-Agent: curl/7.83.1
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Date: Mon, 21 Nov 2022 14:37:34 GMT
< Content-Length: 51
< Content-Type: text/plain; charset=utf-8
< 
{ [51 bytes data]
100    51  100    51    0     0  21860      0 --:--:-- --:--:-- --:--:-- 51000
* Connection #0 to host 127.0.0.1 left intact
*
```